runtime/browser/linux/xwalk_notification_manager.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef XWALK_RUNTIME_BROWSER_LINUX_XWALK_NOTIFICATION_MANAGER_H_
+#define XWALK_RUNTIME_BROWSER_LINUX_XWALK_NOTIFICATION_MANAGER_H_
+
+#include <libnotify/notification.h>
+#include <libnotify/notify.h>
+
+#include <map>
+
+#include "base/callback.h"
+#include "base/containers/scoped_ptr_hash_map.h"
+#include "base/strings/string16.h"
+
+class GURL;
+class SkBitmap;
+
+namespace content {
+class BrowserContext;
+class DesktopNotificationDelegate;
+class RenderFrameHost;
+struct PlatformNotificationData;
+}
+
+namespace xwalk {
+
+class XWalkNotificationManager {
+ public:
+  XWalkNotificationManager();
+  virtual ~XWalkNotificationManager();
```

Comment:
You did not get @pozdnyakov 's meaning. If you do not plan this class as parent, please do not make destructor as virtual.